### ISSUE TITLE
[High] : Many pages: Chrome + Talkback: Currency dropdown is not accessible using screen reader.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8558]
## Action Performed:
1. Using Chrome, open URL staging.new.expensify.com
2. Navigate to Setting using swipe gesture and press double tap on it.
3. Navigate to any workspace using swipe gesture and press double tap on it.
4.  Navigate to general settings using swipe gesture and press double tap on it.
5. Navigate to currency drop down using swipe gesture and press double tap to activate.
6. Observe the behavior.

## Expected Result:
Currency dropdown should be accessible using screen reader.

## Actual Result:
Currency dropdown button is not accessible using screen reader. 

**Other Occurrences:**
Similar issues repro on PR #8196, #9118

## Workaround:
Yes

## Area issue was found in:
General settings

## Failed WCAG checkpoints
2.1.1

## User impact:
Screen reader user may have difficulty accessing Currency dropdown if the Currency dropdown is not accessible.

## Suggested resolution:
Ensure that the Currency dropdown can be accessible using Enter key when screen reader is enabled.                         
Refer to: https://www.w3.org/WAI/WCAG21/Techniques/general/G202

## Platform:
- Android

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/162374464-9e2143d2-d4e4-41f5-95e9-5e215f031a28.mp4
